### ISSUE TITLE
[BUG] Scroll bars missing in output window in eXide 3.0.0

### ISSUE THREAD

**Issue by vincentml** [eXist-db/eXide#377]
After upgrading to eXide version 3.0.0 the output window no longer scrolls when displaying a large set of results. This is a minor inconvenience that I can live with, but this looks like a regression since the previous version of eXide so I'm reporting it as a bug. Previously, the output window would have vertical and horizontal scrollbars as needed. It looks like this can be fixed in CSS but I couldn't quite pinpoint the location of the problem.

Steps to reproduce:

1. Run a query that produces a long set of results. For example:  ```for $i in (1 to 20) return <text>{string-join((1 to 100), ' ')}</text>```
2. The output window displays only the results that fit on screen and is missing the scroll bars

Screenshot of eXide 3.0.0 showing that scroll bars are missing on the output window:

![image](https://user-images.githubusercontent.com/5331729/137567320-de2215bd-6be6-4677-8ad3-560a2377bb66.png)

Screenshot of eXide 2.4.9 showing that there was a scroll bar on the output window:

![image](https://user-images.githubusercontent.com/5331729/137567400-5b4ac6e8-ac0f-4d16-b06c-802648f328e2.png)

Context:

* OS: Windows 2012 R2
* eXist-db Version: 5.3.0
* Java version: Zulu 11.0.6
* eXide app version: 3.0.0

**Comment by joewiz** [comment]
This is a dupe of #325 and was fixed in #358. The fix just hasn't been issued in a new release. I'm working on getting that release out. It's waiting on the merger of another PR: https://github.com/eXist-db/eXide/pull/374. Hopefully in the next few days.

**Comment by vincentml** [comment]
Thanks @joewiz ! I did look to see if there was an already an issue about this problem but I guess I didn't see it because it was already closed.

**Comment by joewiz** [comment]
Great, and sorry! The new release is now out.

**Comment by vincentml** [comment]
Wonderful! Thank you!
